deepinfra/models/base/automatic_speech_recognition.py

```diff
@@ -26,4 +26,4 @@ def generate(self, body) -> AutomaticSpeechRecognitionResponse:
         response = self.client.post(
             form_data, {"headers": {"content-type": form_data.content_type}}
         )
-        return AutomaticSpeechRecognitionResponse(**response.json())
+        return response.json()
```

Comment:
Why do you remove the conversion to the ASRResponse? Now the annotation is wrong (this is not C++, it won't auto-call the constructor or anything).

Reply:
This code works inconsistenly before 3.10. It is related with kw_only keyword.
What about migrating our request and response schemas from dataclasses to pydantic?
Yes, it does come up with a dependency, but, client library will work more robustly across different Python versions.